app/gui2/src/components/GraphEditor.vue

```diff
@@ -77,7 +86,7 @@ function updateNodeContent(id: ExprId, range: ContentRange, content: string) {
         @delete="graphStore.deleteNode(id)"
         @updateExprRect="updateExprRect"
```

Comment:
didn't notice before but *maybe* these could be `kebab-case` instead - i noticed you changed the bindings above to `kebab-case`

Reply:
I prefer to use `camelCase` for emits, an old habit I guess. The linter now actually checks that. Do you think it's a bad idea?

Reply:
i don't really have an opinion tbh, as long as it's consistent then whatever works, works